homeassistant/components/sensor/wunderground.py

```diff
@@ -192,18 +216,19 @@ def update(self):
 class WUndergroundData(object):
     """Get data from WUnderground."""
 
-    def __init__(self, hass, api_key, pws_id=None):
+    def __init__(self, hass, api_key, pws_id=None, lang='EN'):
```

Comment:
You already have a default value for both pws_id and lang in voluptuous. These are not needed here.